operator-framework-core/src/main/java/io/javaoperatorsdk/operator/api/reconciler/ErrorStatusHandler.java

```diff
@@ -0,0 +1,27 @@
+package io.javaoperatorsdk.operator.api.reconciler;
+
+import io.fabric8.kubernetes.client.CustomResource;
+
+public interface ErrorStatusHandler<T extends CustomResource<?, ?>> {
```

Comment:
Once we have the `HasStatus` (or similar) interface, this probably should use that instead of `CustomResource`

Reply:
It will be moved to HasMetada if the other PR is merged.